src/commands/Productivity/remind.ts

```diff
@@ -44,30 +45,40 @@ export default class extends SteveCommand {
 
 		await this.client.schedule.createReminder(duration, msg.author.id, reminder, msg.channel instanceof TextChannel && reminderChannel ? reminderChannel : msg.channel.id);
 
-		return msg.channel.send(msg.language.tget('commandRemindCreated', friendlyDuration(duration)));
+		return msg.channel.send(msg.language.tget('commandRemindCreated', this.getTimeUntilRemind(duration)));
 	}
 
 	public async view(msg: KlasaMessage): Promise<Message> {
-		let output = '';
 		const reminders = this.client.schedule.getUserReminders(msg.author.id);
 		if (reminders.length < 1) throw msg.language.tget('commandRemindNoReminders');
 
-		for (let i = 0; i < reminders.length; i++) {
-			const reminder = reminders[i];
-			const display = await this.getReminderDisplayContent(msg, reminder);
-			output += `**${i + 1}**: ${display} (${friendlyDuration(reminder.time.getTime() - Date.now())} left!)\n\n`;
-		}
-
-		const embedData = msg.language.tget('commandRemindViewEmbed');
+		const response = await msg.send(new MessageEmbed()
+			.setDescription('Loading...')
+			.setColor(msg.author.settings.get(UserSettings.EmbedColor) as ColorResolvable || 0xadcb27));
 
-		const embed = new MessageEmbed()
-			.attachFiles(['./assets/images/alarmclock.png'])
+		const EMBED_DATA = msg.language.tget('commandRemindViewEmbed');
```

Comment:
embedData instead of EMBED_DATA

Reply:
Resolved in [d2bc1b4](https://github.com/stevebot-project/steve/pull/65/commits/d2bc1b4e01002201fb5bc3a40c71f8d25fd2fc7f)